### PR TITLE
[#2] Scaffolding of Webhook application

### PR DESCRIPTION
**Pull request by Satendra-SR**
#2

### Types of changes

- [ ] Bug fix (a change that fixes an issue)
- [x] New feature (a change that adds functionality)

### Short description of what this resolves:

- Added Project scaffolding for webhook handler

### Checklist:

- [x] I have performed a self-review of my own code.
- [x] The code follows the style guidelines of this project.
- [ ] I have documented my code wherever required.
- [ ] The changes requires a change to the documentation.
- [ ] I have updated the documentation based on the my changes.

### CODE REVIEW

**Review by vivek-kumar-pandey — APPROVED** [review]
Looks Good

File: package.json [hunk 1]
```diff
@@ -0,0 +1,10 @@
+{
+  "dependencies": {
+    "@google-cloud/functions-framework": "^2.1.0",
+    "@google-cloud/tasks": "^3.0.2",
+    "pre-commit": "^1.2.2"
+  },
+  "scripts": {
+    "start": "functions-framework functions-framework --target=handle-webhook --port=8080"
+  }
+}
```
Comment (GauravGusain98) [inline package.json]: Why we are not going with the requirements.py method?

File: package.json [hunk 2]
```diff
@@ -0,0 +1,9 @@
+{
+  "dependencies": {
+    "@google-cloud/functions-framework": "^2.1.0",
+    "@google-cloud/tasks": "^3.0.2"
+  },
+  "scripts": {
+    "start": "functions-framework functions-framework --target=handle-webhook --port=8080"
+  }
+}
```
Comment (GauravGusain98) [inline package.json:9]: Why we are not going via the requirements? py method

Comment (Satendra-SR) [reply package.json:9]: Hey, we have done the POC in nodejs, so continue using it for now

**Review by GauravGusain98 — APPROVED** [review]
LGTM

**Pull request merged**